package.json

```diff
@@ -62,7 +62,8 @@
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
     "heroku-serve": "node heroku-server.js",
-    "heroku-postbuild": "cross-env REACT_APP_MOCK=true npm run build"
+    "heroku-postbuild": "cross-env REACT_APP_MOCK=true npm run build",
+    "uu": "nav-ally -f wcag.yml -r yes"
```

Comment:
Er det denne kommandoen for å starte uu-test? `npm run uu` Isåfall hvorfan startes den? Kun lokalt eller i heroku bygget?

Reply:
Du starter UU-testen med `npm run uu`. Da kjøres testen mot Heroku instansen.